src/tribler/core/components/restapi/restapi_component.py

```diff
@@ -150,6 +150,9 @@ def report_callback(reported_error: ReportedError):
     async def shutdown(self):
         await super().shutdown()
 
+        if self._events_endpoint:
+            await self._events_endpoint.shutdown()
```

Comment:
Can you add an explanation why EventsEndpoint requires an explicit shutdown here while all other endpoints do not need it?

Reply:
`EventsEndpoint` inherited from `TaskManager` and the others are not.